drivers/i2c/Kconfig.mspm0

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Texas Instruments
+
+menuconfig I2C_MSPM0
+	bool "TI I2C MSPM0 driver"
+	default y
+	depends on DT_HAS_TI_MSPM0_I2C_ENABLED
+	select PINCTRL
+	help
+	  Enable support for I2C on the TI MSPM0 series.
+
+if I2C_MSPM0
+
+config I2C_CONTROLLER_TIMEOUT
+	bool "I2C Controller Timeout"
+	default y
+	help
+	  Enable support for timeout feature of device
+
+config I2C_MSPM0_MULTI_TARGET_ADDRESS
```

Comment:
See the comment in dts bindings, if target register handled in driver, we don't need it